tests/wpt/web-platform-tests/html/infrastructure/safe-passing-of-structured-data/structured_clone_blob.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<html>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+async_test(function(t) {
+  var blob = new Blob(['<a id="a"><b id="b">hey!</b></a>'], {type:"text/plain"});
+  window.addEventListener("message", this.step_func(function(msg) {
+      assert_equals(msg.data.size, blob.size);
```

Comment:
It probably makes sense to check that it's a Blob first, otherwise this check is likely to fail in a slightly more confusing fashion.